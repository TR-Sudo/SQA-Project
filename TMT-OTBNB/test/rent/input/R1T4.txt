login
PS_user01
rent
logout
quit