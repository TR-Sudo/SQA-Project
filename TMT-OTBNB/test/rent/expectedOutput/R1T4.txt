Enter transaction: Enter username: You have been logged in
Enter transaction: User can not be post standard
Enter transaction: You have been logged out
Enter transaction: 